### PR TITLE
chore: Move from Twitter to X

### LINKED ISSUE #48408: Replace Twitter with X
In the personal setting the footer still links to Twitter, should be replaced with X.
Also the icon is wrong and should be replaced with the X icon.

image

### PR DESCRIPTION
**Pull request by nfebe**
## Description

Update twitter logo

Resolves : https://github.com/nextcloud/server/issues/48408

## Screenshots

| Before | After |
| ---------- | ------- |
|Screenshot from 2024-10-02 10-39-14|Screenshot from 2024-10-02 10-35-13|

### CODE REVIEW

File: apps/settings/templates/settings/personal/development.notice.php [hunk 1]
```diff
@@ -30,43 +30,43 @@
...
+				'<svg aria-hidden="true" width="50" height="50" version="1.1" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg"><circle cx="8" cy="8" r="8" fill="var(--color-text-maxcontrast)"/><path d="M 2.6425781 1.8 C 2.2543781 1.8 2.0276281 2.2362184 2.2488281 2.5558594 L 5.8417969 7.6900156 L 2.2347656 11.9074218 C 2.0013656 12.1798212 2.1947156 12.6 2.5535156 12.6 L 2.9519531 12.6 C 3.1279531 12.6 3.2949844 12.5228622 3.4089844 12.3890624 L 6.5519531 8.7065156 L 8.9179688 12.0878908 C 9.1339688 12.408984 9.5099768 12.6 10.0011718 12.6 L 11.9015628 12.6 C 12.2897628 12.6 12.5173128 12.163312 12.2953128 11.845312 L 8.5042968 6.4195156 L 11.9402344 2.3941408 C 12.139431 2.1601408 11.973215 1.8 11.6651601 1.8 L 11.1585938 1.8 C 10.9835938 1.8 10.816734 1.876453 10.702734 2.0197656 L 7.7982656 5.4058438 L 5.6367188 2.3132812 C 5.4145313 1.9913687 4.9541875 1.8 4.6535156 1.8 L 2.6425781 1.8 z" fill="var(--color-main-background)"/></svg>',
```
Comment (Fenn-CS) [inline apps/settings/templates/settings/personal/development.notice.php:51]: @JuliaKirschenheuter @marcoambrosini any tips on how to properly generate the path in that it centralizes itself in the circle path?

Maybe a tool? Too bad I can't write in svg :grin:

Comment (provokateurin) [inline apps/settings/templates/settings/personal/development.notice.php:51]:
```suggestion
				'<svg aria-hidden="true" width="50" height="50" version="1.1" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg"><circle cx="8" cy="8" r="8" fill="var(--color-text-maxcontrast)"/><path d="m 3.384891,2.6 c -0.3882,0 -0.61495,0.4362184 -0.39375,0.7558594 L 6.5841098,8.4900156 2.9770785,12.707422 C 2.7436785,12.979821 2.9370285,13.4 3.2958285,13.4 H 3.694266 c 0.176,0 0.3430313,-0.07714 0.4570313,-0.210938 L 7.294266,9.5065156 9.6602817,12.887891 C 9.8762817,13.208984 10.25229,13.4 10.743485,13.4 h 1.900391 c 0.3882,0 0.61575,-0.436688 0.39375,-0.754688 L 9.2466097,7.2195156 12.682547,3.1941408 C 12.881744,2.9601408 12.715528,2.6 12.407473,2.6 h -0.506566 c -0.175,0 -0.34186,0.076453 -0.45586,0.2197656 L 8.5405785,6.2058438 6.3790317,3.1132812 C 6.1568442,2.7913687 5.6965004,2.6 5.3958285,2.6 Z" fill="var(--color-main-background)"/></svg>',
```
Centered it with Inkscape

Comment (provokateurin) [reply apps/settings/templates/settings/personal/development.notice.php:51]:
```suggestion
				'<svg aria-hidden="true" width="50" height="50" version="1.1" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg"><circle cx="8" cy="8" r="8" fill="var(--color-text-maxcontrast)"/><path d="m 3.384891,2.6 c -0.3882,0 -0.61495,0.4362184 -0.39375,0.7558594 L 6.5841098,8.4900156 2.9770785,12.707422 C 2.7436785,12.979821 2.9370285,13.4 3.2958285,13.4 H 3.694266 c 0.176,0 0.3430313,-0.07714 0.4570313,-0.210938 L 7.294266,9.5065156 9.6602817,12.887891 C 9.8762817,13.208984 10.25229,13.4 10.743485,13.4 h 1.900391 c 0.3882,0 0.61575,-0.436688 0.39375,-0.754688 L 9.2466097,7.2195156 12.682547,3.1941408 C 12.881744,2.9601408 12.715528,2.6 12.407473,2.6 h -0.506566 c -0.175,0 -0.34186,0.076453 -0.45586,0.2197656 L 8.5405785,6.2058438 6.3790317,3.1132812 C 6.1568442,2.7913687 5.6965004,2.6 5.3958285,2.6 Z" fill="var(--color-main-background)"/></svg>',
```
Centered it with Inkscape

File: apps/settings/templates/settings/personal/development.notice.php [hunk 2]
```diff
@@ -30,43 +30,43 @@
...
 				'<a target="_blank" aria-label="{facebooktext}" rel="noreferrer noopener" href="https://www.facebook.com/Nextclouders/">',
-				'<a target="_blank" aria-label="{twittertext}" rel="noreferrer noopener" href="https://twitter.com/nextclouders">',
+				'<a target="_blank" aria-label="{xtext}" rel="noreferrer noopener" href="https://x.com/nextclouders">',
 				'<a target="_blank" aria-label="{mastodontext}" rel="noreferrer noopener" href="https://mastodon.xyz/@nextcloud">',
 				'<a target="_blank" aria-label="{rsstext}" rel="noreferrer noopener" href="https://nextcloud.com/blog/">',
 				'<a target="_blank" aria-label="{mailtext}" rel="noreferrer noopener" href="https://newsletter.nextcloud.com/?p=subscribe&amp;id=1">',
 				'</a>',
 				$l->t('Like our Facebook page'),
-				$l->t('Follow us on Twitter'),
+				$l->t('Follow us on x'),
```
Comment (provokateurin) [inline apps/settings/templates/settings/personal/development.notice.php:62]:
```suggestion
				$l->t('Follow us on X'),
```

**Review by provokateurin — CHANGES_REQUESTED** [review]

File: apps/settings/templates/settings/personal/development.notice.php (hunk 1 above)
Comment (Fenn-CS) [reply apps/settings/templates/settings/personal/development.notice.php:51]: thanks @provokateurin looks way better.

**Review by szaimen — APPROVED** [review]
LGTM but didnt test

**Review by provokateurin — APPROVED** [review]

**Pull request merged**